lib/_http_incoming.js

```diff
@@ -114,19 +114,21 @@ IncomingMessage.prototype._read = function _read(n) {
     readStart(this.socket);
 };
 
-
 // It's possible that the socket will be destroyed, and removed from
 // any messages, before ever calling this.  In that case, just skip
 // it, since something else is destroying this connection anyway.
-IncomingMessage.prototype.destroy = function destroy(error) {
-  // TODO(ronag): Implement in terms of _destroy
-  this.destroyed = true;
-  if (this.socket)
-    this.socket.destroy(error);
-  return this;
+IncomingMessage.prototype._destroy = function _destroy(err, cb) {
```

Comment:
I was wondering if using the standard destroy implementation could cause some performance regressions since it has more checks than the logic that was in place before.

Reply:
Is there any specific benchmark you would like me to run to check this?

Reply:
I don't think we need to worry about destroy performance.